### DISCUSSION TITLE
Netdata installation gets stuck

### DISCUSSION THREAD

**Discussion by iodize6399** [tteck/Proxmox#2272] [Help Needed]
![image](https://github.com/tteck/Proxmox/assets/4197707/f145d4c1-2465-40d4-9fe5-3f0e33fdc42a)

**Comment by tteck** [comment]
The script runs without any problems from this end. 

```
    _   __     __  ____        __
   / | / /__  / /_/ __ \____ _/ /_____ _
  /  |/ / _ \/ __/ / / / __ `/ __/ __ `/
 / /|  /  __/ /_/ /_/ / /_/ / /_/ /_/ /
/_/ |_/\___/\__/_____/\__,_/\__/\__,_/

 ✓ Set up repository
 ✓ Installed Netdata
 ✓ Completed Successfully!


 Netdata should be reachable at http://192.168.86.3:19999 

root@server:~# 
```

**Reply by iodize6399** [reply]
Verbose
![image](https://github.com/tteck/Proxmox/assets/4197707/2ad9f050-846a-48fe-93bd-d0a77b55686e)

I believe I need to install post-install script to remove the enterprise repositories. However this is not the requirement for other scripts which run perfect without the post install script.

**Reply by tteck** [reply]
~All scripts perform an update/upgrade, you should encounter the same issue.~

**Reply by tteck** [reply]
My bad, it only affects the Proxmox host.
